src/SME.SGP.Aplicacao/Consultas/ConsultasNotasConceitos.cs

```diff
@@ -191,9 +191,9 @@ public async Task<NotasConceitosRetornoDto> ListarNotasConceitos(ListaNotasConce
                         else
                         {
                             IEnumerable<ComponenteCurricularEol> disciplinasRegenciaEol = await servicoEOL.ObterComponentesCurricularesPorCodigoTurmaLoginEPerfilParaPlanejamento(filtro.TurmaCodigo, servicoUsuario.ObterLoginAtual(), servicoUsuario.ObterPerfilAtual());
-                            if (disciplinasRegenciaEol == null || !disciplinasRegenciaEol.Any())
+                            if (disciplinasRegenciaEol == null || !disciplinasRegenciaEol.Where(x => x.Regencia).Any())
```

Comment:
assim também funciona
```suggestion
                            if (disciplinasRegenciaEol == null || !disciplinasRegenciaEol.Any(x => x.Regencia))
```